nilearn/plotting/js_plotting_utils.py

```diff
@@ -165,7 +165,7 @@ def open_in_browser(self, file_name=None, temp_file_lifetime=30):
 
         """
         if file_name is None:
-            fd, file_name = tempfile.mkstemp('.html', 'nilearn_surface_plot_')
+            fd, file_name = tempfile.mkstemp('.html', 'nilearn_plot_')
```

Comment:
Use a context manager here.

Reply:
no because we don't want to close the file when we exit the context, or even when the function returns. here I'm just making the page's title more generic, not changing the logic of `open_in_browser`

Reply:
Ok